src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -0,0 +1,94 @@
+{#**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<tr id="orderProduct_{{ product.orderDetailId }}">
+  <td>
+    {% if product.imagePath %}
+      <img src="{{ product.imagePath }}" alt="{{ product.name }}">
+    {% endif %}
+  </td>
+  <td>
+    <a href="{{ path('admin_product_form', {'id': product.id}) }}">
+      <p class="mb-0">{{ product.name }}</p>
+      {% if product.reference %}
+        <p class="mb-0">
+          {{ 'Reference number:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+          {{ product.reference }}
+        </p>
+      {% endif %}
```

Comment:
That would be cool to add some space between elements which are not at the end of a parent element to make it more readable

Reply:
:ballot_box_with_check: Thanks